lua/neogen/generator.lua

```diff
@@ -210,6 +210,17 @@ return setmetatable({}, {
             end
         end
 
+        if as_snippet then
+          local ph = 1
+          for i, line in ipairs(content) do
```

Comment:
couldn't you just use `i` instead of `ph` here?

Reply:
but you don't check for this

Reply:
@mg979  ^^

Reply:
There I was adding placeholders only if the line ended with an empty space. I changed it by now to use marks positions anyway.